assets/tests/factories/notification.ts

```diff
@@ -84,3 +85,23 @@ export const bridgeLoweredNotificationFactory = Factory.define<
   state: "unread",
   content: bridgeLoweredNotificationContentFactory.build(),
 }))
+
+const detourActivatedNotificationContentFactory =
+  Factory.define<DetourNotification>(() => ({
+    $type: NotificationType.Detour,
+    status: "activated",
+    detourId: 79,
+    headsign: "79 - Wooded Mountains",
+    route: "79",
+    direction: "Inbound",
+    origin: "Maple Knoll",
```

Comment:
question: is it possible to not hardcode these values?

Reply:
It's a bit of a struggle when tests assume hardcoded values from factories and then in the future we decide to make them generated, so I'd prefer to start with generated values.

I'd be open to pulling in faker @mbta/skate-developers

Reply:
I agree - I did this just to get _something_ working, but I'd rather get non-hardcoded values in before merging.

Reply:
This was done in https://github.com/mbta/skate/pull/2819